data/A10-g.json

```diff
@@ -14,24 +14,70 @@
   ],
   "links": [
     {
-      "name": "Follow me on Twitter",
+      "name": "Engage with me on Twitter",
       "url": "https://twitter.com/Abhishek_G101",
       "icon": "FaTwitter"
     },
     {
-      "name": "Follow me on LinkedIn",
+      "name": "Connect with me on LinkedIn",
       "url": "https://www.linkedin.com/in/abhishek-gupta-95275b233",
       "icon": "FaLinkedin"
     },
     {
-      "name": "Follow me on Github",
+      "name": "Follow my Open Source Journey on Github",
       "url": "https://github.com/A10-g",
       "icon": "FaGithub"
     },
     {
-      "name": "Follow me on Hashnode",
+      "name": "Take a glance of my writings on Hashnode",
       "url": "https://abhishekgupta10.hashnode.dev",
       "icon": "SiHashnode"
     }
+  ],
+  "milestones":[
+    {
+      "title": "Started my Journey in Computer Science",
+      "description": "I enrolled to 4 year Undergraduate degree in Computer Engineering.",
+      "date": "January 2021",
+      "icon": "FaLaptop",
+      "color": "#1e3530"
+    },
+    {
+      "title": "Started Coding",
+      "description": "I began learning to code and solve Dsa questions on Leetcode and GeeksforGeeks",
+      "date": "February 2022",
+      "icon": "FaCode",
+      "color": "#1e3530"
+    }, 
+    {
+      "title": "The Beginning to Open Source",
+      "description": "I began my Open Source journey on GitHub.",
+      "date": "September 2022",
+      "icon": "FaGithub",
+      "color": "#1e3530"
+    }, 
+    {
+      "title": "First Hactoberfest",
+      "description": "I contributed to my first Hactoberfest",
+      "date": "October 2022",
+      "icon": "FaCode",
+      "color": "#1e3530"
+    },
+    {
+      "title": "Started Engaging on socials",
+      "description": "I started engaging with folks on twitter ",
+      "date": "January 2023",
+      "icon": "SiTwitter",
+      "color": "#1e3530",
+      "url":"https://twitter.com/Abhishek_G101"
+    },
+    {
+       "title": "First Blog",
+      "description": "Contributed to my first Hactoberfest",
```

Comment:
hey @A10-g, I think you added the wrong description kindly look into it & change it. ✌️
 ``` suggestion
      "description": "Started my own blogs",
```